sequences/seqs.gobra

```diff
@@ -0,0 +1,458 @@
+/*
+  This file is part of gobra-libs which is released under the MIT license.
+  See LICENSE or go to https://github.com/viperproject/gobra-libs/blob/main/LICENSE
+  for full license details.
+
+  This file is inspired by the standard libraries and axiomatisations of the following verifiers:
+  - dafny-lang/libraries: https://github.com/dafny-lang/libraries/blob/master/src/Collections/Sequences/Seq.dfy
+*/
+
+// This package defines lemmas for sequences commonly used in specifications.
+package seqs
+
+import utils "utils"
```

Comment:
If I use this, I get an error of the form: got unknown identifier utils.

Reply:
I tried this again. Instead of getting "got unknown identifier utils", I get:
```
12:39:29.028 [main] ERROR viper.gobra.GobraRunner$ - An assumption was violated during execution.
12:39:29.032 [main] ERROR viper.gobra.GobraRunner$ - Logic error: This case should be unreachable, but got unknown
viper.gobra.util.Violation$LogicException: Logic error: This case should be unreachable, but got unknown
	at viper.gobra.util.Violation$.violation(Violation.scala:27)
	at viper.gobra.frontend.info.implementation.typing.ExprTyping.getTypeFromCtxt(ExprTyping.scala:887)
	at viper.gobra.frontend.info.implementation.typing.ExprTyping.wellDefActualExpr(ExprTyping.scala:445)
	...
```
Turns out that the package name was not `utils`, but rather `util`. I have no idea why everything worked until now.